msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCasePerf.java

```diff
@@ -0,0 +1,164 @@
+import com.microsoft.identity.client.ui.automation.interaction.PromptHandlerParameters;
+import com.microsoft.identity.client.ui.automation.interaction.PromptParameter;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.AadPromptHandler;
+import com.microsoft.identity.client.ui.automation.logging.appender.FileAppender;
+import com.microsoft.identity.client.ui.automation.logging.formatter.SimpleTextFormatter;
+import com.microsoft.identity.client.ui.automation.utils.CommonUtils;
+import com.microsoft.identity.common.CodeMarkerManager;
+import com.microsoft.identity.common.internal.controllers.CommandDispatcherHelper;
+import com.microsoft.identity.internal.testutils.labutils.LabConfig;
+import com.microsoft.identity.internal.testutils.labutils.LabConstants;
+import com.microsoft.identity.internal.testutils.labutils.LabUserQuery;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+// Silent Auth with force_refresh
+// https://identitydivision.visualstudio.com/DefaultCollection/IDDP/_workitems/edit/99563
+public class TestCasePerf extends AbstractMsalUiTest {
+
+    @Test
+    public void test_99563() {
+        final TokenRequestLatch latch = new TokenRequestLatch(1);
+
+        final AcquireTokenParameters parameters = new AcquireTokenParameters.Builder()
+                .startAuthorizationFromActivity(mActivity)
+                .withLoginHint(mLoginHint)
+                .withScopes(Arrays.asList(mScopes))
+                .withCallback(successfulInteractiveCallback(latch))
+                .withPrompt(Prompt.SELECT_ACCOUNT)
+                .build();
+
+
+        final InteractiveRequest interactiveRequest = new InteractiveRequest(
+                mApplication,
+                parameters,
+                new OnInteractionRequired() {
+                    @Override
+                    public void handleUserInteraction() {
+                        final String username = mLoginHint;
+                        final String password = LabConfig.getCurrentLabConfig().getLabUserPassword();
+
+                        final PromptHandlerParameters promptHandlerParameters = PromptHandlerParameters.builder()
+                                .prompt(PromptParameter.SELECT_ACCOUNT)
+                                .loginHint(mLoginHint)
+                                .sessionExpected(false)
+                                .consentPageExpected(false)
+                                .speedBumpExpected(false)
+                                .build();
+
+                        new AadPromptHandler(promptHandlerParameters)
+                                .handlePrompt(username, password);
+                    }
+                }
+        );
+
+        interactiveRequest.execute();
+        latch.await(TokenRequestTimeout.SHORT);
+
+        final IAccount account = getAccount();
+
+        CodeMarkerManager.setEnableCodeMarker(true);
+        final int numberOfOccurrenceOfTest = 10;
```

Comment:
This could probably move at the top of the file

Reply:
Done. Thanks for the suggestion.